src/table.ts

```diff
@@ -895,6 +895,10 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
         reqOpts.rowsLimit = rowsLimit - rowsRead;
       }
 
+      if (!reqOpts.rows && lastRowKey) {
+        return
```

Comment:
This will just leave the userStream hanging indefinitely, causing the caller's application to hang